### DISCUSSION TITLE
setting mindate as minDate as today

### DISCUSSION THREAD

**Discussion by sanathusk** [pocketbase/pocketbase#2160] [Q&A] [answered]
Is it possible to set the min value for date as today?Basically, i have a due date field, the user should be prevented from setting the due date to a date previous to current date, i am currently have a check on the UI , but the user could by pass this restriction.How can i put this restriction?

**Comment by ganigeorgiev** [comment, marked as answer]
You can set a dynamic min date constraint (including "today" with `@now`) using Create API rule like:

```js
@request.data.dueDate >= @now
```

If you want to enforce every `dueDate` change to be at greater than or equal to the last date, you can define an Update API rule like:

```js
@request.data.dueDate:isset = false || @request.data.dueDate >= dueDate
```

**Reply by sanathusk** [reply]
Thanks @ganigeorgiev
